src/blocks/taskwarrior.rs

```diff
@@ -80,10 +83,20 @@ pub struct TaskwarriorConfig {
 
     /// Format override if the count is zero
     pub format_everything_done: FormatTemplate,
+
+    /// Data directory. Defaults to ~/.task but it's configurable in taskwarrior
+    /// (data.location in .taskrc) so make it configurable here, too
+    pub data_location: String,
 }
 
 impl Default for TaskwarriorConfig {
     fn default() -> Self {
+        let home_dir = match dirs::home_dir() {
+            Some(path) => path.into_os_string().into_string().unwrap(),
+            None => "".to_owned(),
+        };
+        let task_dir = format!("{}/.task", home_dir);
```

Comment:
`shellexpand` will expand `~` as a home directory anyway, so I suppose you can just do `let task_dir = "~/.task"`.

Reply:
> `shellexpand` will expand `~` as a home directory anyway, so I suppose you can just do `let task_dir = "~/.task"`.

That's better. Done.